helm/_system/nginx-ingress/daemonset.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    name: nginx-ingress-lb
+  name: nginx-ingress-lb
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      name: nginx-ingress-lb
+  template:
+    metadata:
+      labels:
+        name: nginx-ingress-lb
+      annotations:
+        pod.beta.kubernetes.io/init-containers: '[
+          {
+          "name": "sysctl",
+            "image": "busybox",
+            "imagePullPolicy": "IfNotPresent",
+            "command": ["sh", "-c", "sysctl -w net.core.somaxconn=32768; sysctl -w net.ipv4.ip_local_port_range=\"1024 65535\""],
+            "securityContext": {
+              "privileged": true
+            }
+          }
+        ]'
+    spec:
+      hostNetwork: true
+      containers:
+      - args:
+        - /nginx-ingress-controller
+        - --ingress-class=service
+        - --default-backend-service=$(POD_NAMESPACE)/default-http-backend
+        - --configmap=$(POD_NAMESPACE)/nginx-ingress-lb
```

Comment:
Put this here:

```
- --watch-namespace=gaia
```

Or find a way to automatically configure `gaia` here. Hint: `$(POD_NAMESPACE)` does not work. :)